test/test.js

```diff
@@ -139,33 +147,101 @@ describe('rollup-plugin-babel', function() {
 	});
 
 	it('allows transform-runtime to be used instead of bundled helpers', () => {
-		let warnCalled = false;
+		const warnings = [];
 		return bundle(
 			'samples/runtime-helpers/main.js',
 			{ runtimeHelpers: true },
 			{},
 			{
-				onwarn: function(warning) {
-					assert.equal(warning.code, 'UNRESOLVED_IMPORT');
-					assert.equal(warning.source, '@babel/runtime/helpers/classCallCheck');
-					warnCalled = true;
+				onwarn(warning) {
+					warnings.push(warning.message);
 				},
 			},
 		).then(({ code }) => {
-			assert.ok(warnCalled, 'onwarn was never triggered about unresolved imports');
-			assert.ok(!~code.indexOf(HELPERS));
+			assert.deepStrictEqual(warnings, [
+				`'@babel/runtime/helpers/classCallCheck' is imported by samples${path.sep}runtime-helpers${path.sep}main.js, but could not be resolved – treating it as an external dependency`,
```

Comment:
Couldnt u normalize separator in Rollup? I think many tools do that, seems to simplify stuff when people integrate with it (like sort of in here)

Reply:
I guess we should do that at some point, but right now, this would be a massive breaking change that I am very afraid of. There is a lot to consider, and this needs to be communicated well, and we do need to do a lot of testing on Windows. Maybe something to tackle for Rollup@3.

Reply:
Out of curiosity - Rollup@3? 😄I guess this won't happen in the near future 😉

Reply:
There is more-or-less a solid set of breaking changes I would love to come in Rollup@2 which I would not want to extend—actually I think it is already too long. Yes, not too soon.